### DISCUSSION TITLE
Trouble with adding files after installing Git

### DISCUSSION THREAD

**Discussion by TabW** [community/community#23838] [New to GitHub]
<p>After successfully installing Git to my computer, I opened the Git Bash and typed in some test command lines to see if everything is working. The command line I typed was “git add .” and, I was given an output of errors.</p>
<p>This is the error I get:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/6/f651bd57be72e5e6ae88435f0511c9731729d830.jpeg" data-download-href="/uploads/short-url/z92Ad7AqWMbTamLGxSRfoMRgUZW.jpeg?dl=1" title="Github_addfile_error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/88566437/181079646-7d1871fb-4a62-44bf-aa73-87061ce3e424.jpeg" alt="Github_addfile_error" data-base62-sha1="z92Ad7AqWMbTamLGxSRfoMRgUZW" width="666" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f651bd57be72e5e6ae88435f0511c9731729d830_2_666x500.jpeg, https://aws1.discourse-cdn.com/github/original/3X/f/6/f651bd57be72e5e6ae88435f0511c9731729d830.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/6/f651bd57be72e5e6ae88435f0511c9731729d830.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/6/f651bd57be72e5e6ae88435f0511c9731729d830_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Github_addfile_error</span><span class="informations">960×720 77.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I tried 5 times to uninstall and reinstall Git and, I also tried changing the install path. In addition, I tried looking up my issue on Google and, I found a post on the Github community with my similar issue, however the issue was not resolved.</p>
<p>If you need more information, please let me know.</p>
<p>Thanks in advanced.</p>

**Comment by airtower-luna** [comment]
<p>That looks like you previously ran <code>git init</code> in your home directory and turned the whole thing into a Git repository. Otherwise <code>git add</code> would complain about being called outside a repository. That’s (almost) certainly not what you want, though: In general you want one repository per piece of software or project.</p>
<p>Before you do the following, make sure you have the right one. There’s no going back (except with backups). :warning: To get rid of the accidental repository, delete its <code>.git/</code> directory. Double check it’s the right one, and not a repository you actually want to keep.</p>

**Comment by TabW** [comment, marked as answer]
<p>I believe the issue is fixed. I used the command line to change the directory of the repository and, placed it in its own folder. Now, I am able to add files.</p>
<p>I am new to Git so I appreciate your help.</p>
